src/utils/jwt.js

```diff
@@ -10,10 +10,24 @@ export const sendTokenResponse = (res, user, message) => {
 
 export const generateToken = (user) => {
   return jwt.sign({ data: user }, process.env.JWT_SECRET, {
-    expiresIn: `${process.env.JWT_EXPIRE}d`
+    expiresIn: `${process.env.JWT_EXPIRE}s`
   });
 };
 
 export const decodeJwtToken = (token) => {
   return jwt.verify(token, process.env.JWT_SECRET);
 };
+
+export const sendRefreshTokenResponse = (res, user, message) => {
```

Comment:
This needs to return both an access token and a refresh token. The same goes for the initial sendTokenResponse function. Both tokens must be returned. We could use the same function for this purpose right, 2 won't be required I guess.